backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/ConfirmationCodeMapper.java

```diff
@@ -0,0 +1,32 @@
+package ca.gov.dtsstn.cdcp.api.service.domain.mapper;
+import java.util.List;
```

Comment:
add a space after the package line

Reply:
should be fixed with the formatter ive used